docs/python/python.rst

```diff
@@ -1,6 +1,12 @@
 Python API Documentation
 ========================
 
+.. only:: not bindings_python
+
+    .. warning::
+
+        This documentation was build while python bindings were disabled. Most probably, this part of the documentation is either empty or outdated. Please enable :code:`BUILD_BINDINGS_PYTHON` in :code:`cmake` and build the documentation again.
```

Comment:
```suggestion
        This documentation was built while python bindings were disabled. This part of the documentation is likely either empty or outdated. Please enable :code:`BUILD_BINDINGS_PYTHON` in :code:`cmake` and build the documentation again.
```

Reply:
Would it be outdated or would it always just be empty?

Reply:
This depends on whether you have already build the python bindings in this particular build folder before. If you did, you may get proper documentation (if you didn't change anything about that), you may get something outdated (if you changed the python bindings, but did not build it), you may get nothing (if something changed that breaks how python imports the lib. I guess this can happen).